code/src/main/java/frc/robot/RobotConfig.java

```diff
@@ -45,4 +46,12 @@
         public final double lengthInCms = 0;
         public final double weightInKgs = 0;
     }
+
+    public final class Intake {
+        public final int intakeControllerChannel = 1;
```

Comment:
i'd recommend changing these names to intakeMotorPort, armMotorPort, armEncoderPort, etc